docker/multi-process/scripts/init

```diff
@@ -5,6 +5,15 @@ export LC_ALL=en_US.UTF-8
 
 cd /app
 
+# Do we have a Rails secret yet?
+# If not, generate one, but persist it for the container's lifetime.
+if grep -q ^APP_SECRET_TOKEN=REPLACE_ME_NOW .env.example; then
```

Comment:
Hmm the condition only checks the `.env.example` file, if a user upgrades the docker container to this version and has `APP_SECRET_TOKEN` set in the environment, the block will execute. We will still replace the random value with the provided `APP_SECRET_TOKEN` later, but as far as I understand it we not need to run the block at all if the secret token is already passed as an environment variable.
It's mainly an optimization but should also avoid confusion by not outputting `Generating random APP_SECRET_TOKEN`